### PR TITLE
Add ethiopic script for amharic and tigrinya

### PR DESCRIPTION
**Pull request by andela-amagana**
Add ethiopic script for use in amharic and tigrinya.

[PR - ws-alephamp-queue-renderer](https://github.com/bbc/ws-alephamp-queue-renderer/pull/358)
[WS2020-578](https://jira.dev.bbc.co.uk/browse/WS2020-578)

### CODE REVIEW

File: lib/settings/scripts/ethiopic/_config.scss [hunk 1]
```diff
@@ -0,0 +1,3 @@
+@if $script == 'ethiopic' {
+
```
Comment (pjlee11) [inline lib/settings/scripts/ethiopic/_config.scss]: Just to note this is empty as the ethopic script is yet to have a typography review but we are expecting this to be filled with overrides.

**Comment by pjlee11** [comment]
This will also need a version bump :)

**Comment by andela-amagana** [comment]
Hello @sareh The reason for creating the placeholder was to avoid having unused CSS loading,
for services that utilize custom fonts and have `latin` set as a placeholder script type.
I'll look into how we can used the shared script type while avoiding having to create placeholder scripts.
Allow us in the meantime, to have this here for the purpose indicted above, of loading the required
custom fonts.
Thanks

**Comment by sareh** [comment]
@andela-amagana Thanks for your reply, Alex. These script versions pull in all the code for the Latin version *plus* the script-specific overrides.  So, if there are no overrides at all and you are using a custom font, you could just use the `latin` version of Grandstand on your page. 

I've also noticed your new PR to replicate this work for the Gujarati script: https://github.com/bbc/grandstand/pull/84 and your [comment on this JIRA ticket](https://jira.dev.bbc.co.uk/browse/WS2020-704?focusedCommentId=3579202&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-3579202)

We can use the Latin version of Grandstand on multiple sites.  News and Sport currently use it for pages. You can use it for Gujarati, Amharic and Tigrinya sites too.

**Pull request merged**